### ISSUE TITLE
Direct Page Break Does Not Work

### ISSUE THREAD

**Issue by d-e-v-esh** [accordproject/web-components#259]
# Bug Report 🐛

When we click on the page-break icon, it adds ``---`` in the markdown panel as the page break. But when we directly write ``---`` on the editor then page break doesn't work. 

## Expected Behavior
Writing ``---`` should  add a page break on the editor

## Current Behavior
Writing ``---`` doesn't create a page break. It just writes it as it is.

## Possible Solution

It would be very convenient if the user doesn't have to touch the mouse and can directly add a page break from the keyboard by entering ``---``. 

If ``---`` gets entered in the middle of the line then the cursor should jump to the next line and make a page break. If the user did this by mistake then he can just backspace once, and we would put him where he was before the page break occurred.

## Steps to Reproduce
1. Write ``---`` on the editor

### Desktop
 - OS: Windows 10
 - Browser: Chrome
 - Version 88.0.4324.182 (Official Build) (64-bit)

**Comment by dselman** [comment]
I've marked this as an enhancement, because in general markdown syntax is not converted to formatted text. You could consider whether it would be more appropriate to add a keyboard hotkey for this instead (consistent with CTRL-B etc used for formatting). Perhaps looking at GDocs or Office 365 will give some inspiration?

**Comment by d-e-v-esh** [comment]
@dselman That is very true. But there is already a keyboard shortcut assigned to that task. When you go and hover over the Page Break icon, it will show that the shortcut for that is  ``ctrl + enter`` but that is not working either for some reason.  I'm looking into it for quite some time, I will try to find a solution if I can.

**Comment by irmerk** [comment]
Pinging @Michael-Grover on this.

**Comment by Michael-Grover** [comment]
I agree with Dan. I think fixing the shortcut is a higher priority than this enhancement. I created an issue for the shortcut bug: https://github.com/accordproject/web-components/issues/263
